src/test/scala/com/ubirch/webui/services/GraphClientMockOk.scala

```diff
@@ -16,7 +16,7 @@ class GraphClientMockOk extends GraphClient {
     r.toList
   }
 
-  override def getUPPs(from: Long, to: Long, hwDeviceId: String): Future[UppState] = ???
+  def getUPPs(from: Long, to: Long, hwDeviceId: String): Future[UppState] = ???
```

Comment:
why do you remove the override here? just out of curiosity. :)

Reply:
With it intellij has some troubles getting the other classes that also implements this def

Reply:
ah, interesting.